### DISCUSSION TITLE
Submission Guidelines: External Dependencies

### DISCUSSION THREAD

**Discussion by ishabhray** [larics/icuas22_competition#59] [Q&A] [answered]
Hello,

We aim to make a submission today. We are not using Docker. Some of our ROS packages have external dependencies (ex. ros-noetic-nlopt).

How do can we ensure the dependencies will be available on the testing environment? Can we submit a script that install all dependencies? However, it would need root access.

**Comment by fpetric** [comment, marked as answer]
We manually check each build that fails and install any missing packages, notify the team and so on. So I would recommend to just submit a solution, we will check and then we can see if there are any 'exotic' dependencies missing.
